_data/simple-icons.json

```diff
@@ -6787,6 +6787,12 @@
             "hex": "11397E",
             "source": "https://www.lot.com/us/en/kaleidoscope-inflight-magazine"
         },
+        {
+            "title": "LTspice",
+            "hex": "900028",
+            "source": "https://www.analog.com/media/en/news-marketing-collateral/solutions-bulletins-brochures/ltspice-keyboard-shortcuts.pdf",
```

Comment:
The actual page where I found the PDF is this: https://www.analog.com/en/design-center/design-tools-and-calculators/ltspice-simulator.html

But it is very difficult to know where the logo can be found by looking at that page and hence I linked the PDF